django_redshift_backend/base.py

```diff
@@ -1395,3 +1398,13 @@ def check_constraints(self, table_names=None):
         No constraints to check in Redshift.
         """
         pass
+
+    @async_unsafe
+    def create_cursor(self, name=None):
+        cursor = super().create_cursor()
+        DJANGO_REDSHIFT_ENABLE_TENANT_SCHEMA = getattr(settings, 'REDSHIFT_SET_TENANT_SCHEMA', False)
+
+        if DJANGO_REDSHIFT_ENABLE_TENANT_SCHEMA:
+            cursor.execute(f"SET SEARCH_PATH TO '{connection.schema_name}', public")
```

Comment:
I don't think this is a feature of the Redshift Backend.
I understand that you can't use ‘django_tenants.postgresql_backend’ provided by django-tenants, but I think you should implement a DatabaseWrapper in your application, as django-tenants does.
https://github.com/django-tenants/django-tenants/blob/c02b5a46ae8abc089c3f5f355b4d116e25572a6a/django_tenants/postgresql_backend/base.py#L60